ci/github/release.go

```diff
@@ -35,7 +35,8 @@ type Releaser struct {
 
 // Release represents a Github release
 type Release struct {
-	Id int64
+	Id      int64
```

Comment:
[Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case. For example, "URL" should appear as "URL" or "url" (as in "urlPony", or "URLPony"), never as "Url". ](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)
```suggestion
	ID      int64
```

Reply:
Thanks, didn't know that